kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/NamespaceVisitFromServerGetWatchDeleteRecreateWaitApplicableListImpl.java

```diff
@@ -242,6 +229,9 @@ public List<HasMetadata> get() {
 
   @Override
   public ListVisitFromServerGetDeleteRecreateWaitApplicable<HasMetadata> inNamespace(String explicitNamespace) {
+    if (explicitNamespace == null) {
```

Comment:
`Utils.isNotNullOrEmpty` instead?

We might also want to implement an alternative `Utils.isNotBlank`

Reply:
It can be - but we aren't doing that check with the other inNamespace locations.  Ideally we would be consistent about not accepting empty / null anywhere a name was expected - so maybe a separate issue to cover that.

Reply:
Creating an issue

Reply:
https://github.com/fabric8io/kubernetes-client/issues/4057